espnet/nets/chainer_backend/asr_transformer/decoders.py

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
```

Comment:
This function must be at `espnet/espnet/nets/chainer_backend/`
How about `espnet/espnet/nets/chainer_backend/decoders_transformer.py`?